user_docs/git_in_OpenSARlab.md

```diff
@@ -0,0 +1,21 @@
+# Git in OpenSARlab
+
+### ASF's Jupyter Notebook Library
+ASF's OpenSARlab Jupyter Notebook library lives in the 
+[asf-jupyter-notebooks GitHub repository](https://github.com/asfadmin/asf-jupyter-notebooks).
+
+A git puller pulls in any changes to the repo each time a user's OpenSARlab server starts up. 
+If a user has made changes to a notebook and the same notebook has been updated by ASF in the 
+asf-jupyter-notebooks repo, the user will end up with two copies of that notebook. The user's
+version of the notebook will have a timestamp appended to its name and the notebook with the 
+original, unaltered name will contain the changes made by ASF.
+
+Please note that we have noticed an occasional bug with the git puller. If a notebook has been
+updated by ASF but you do not receive an updated version after restarting your server, try
+deleting the notebook in question and [manually restart the server](restarting_server_and_kernel.md). 
+
```

Comment:
We need a blurb about fixing broken servers. In particular, if an user tries to start a server and the server errors out and the error mentions something about bad git puller (see screen capture?), use the `General SAR processing (without git puller)` profile.  

Also, users may not have to delete their "bad" notebook. They just need to change it so that git recognizes the change. This can be done by applying a command line `touch` to the file to update the timestamp.

Reply:
I added this issue as a topic in the troubleshooting guide.